lib/voicebase/v2/client.rb

```diff
@@ -79,6 +133,22 @@ def get_media_progress(args = {}, headers = {})
         ), api_version)
       end
 
+      def delete_file(args = {}, headers = {})
+        url = if args[:media_id]
+                uri + "/media/#{args[:media_id]}"
+              else
+                raise ArgumentError, "Missing argument :media_id"
+              end
+
+        response = self.class.delete(
+            url,
```

Comment:
RubyMine formatted it this way, like it did for lines 137..141. Apparently its rule is: double indent when method call arguments are pushed onto their own lines. It does single indents with discrete statements, like if/else/end etc. @JustinAiken do you see a strong reason to change this?